react-tagsinput.js

```diff
@@ -411,8 +411,9 @@
         var tag = this._tag();
         var empty = tag === '';
         var keyCode = e.keyCode;
-        var add = addKeys.indexOf(keyCode) !== -1;
-        var remove = removeKeys.indexOf(keyCode) !== -1;
+        var key = e.key;
+        var add = addKeys.indexOf(keyCode) !== -1 || addKeys.indexOf(key) !== -1;
```

Comment:
I would suggest using `key` as the default and `keyCode` as the fallback given [keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is officially deprecated.

Reply:
Existing code probably uses keyCodes so the minor value of short circuiting probably goes more to keeping it first.